packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts

```diff
@@ -435,7 +435,11 @@ export const useNotifiSubscribe: ({
           .find((f) => f?.filterType === filterType);
         if (filter === undefined || filter.id === null) {
           await deleteThisAlert();
-          return null;
+          throw new Error(
+            `No applicableFilters filter for sources: ${JSON.stringify(
+              sources,
+            )}`,
```

Comment:
do we also want to throw on line 523?

Reply:
Yeah, Also added for single source alert (line#523). Thanks for reminding @juni-notifi